pkg/rkeworker/execute_windows.go

```diff
@@ -167,62 +130,32 @@ func executePowerShell(rootCtx context.Context, scriptPath string, args ...strin
 	g.Go(func() error {
 		if err := command.Run(); err != nil {
 			if exitError, ok := err.(*exec.ExitError); ok {
-				waitStatus := exitError.Sys().(syscall.WaitStatus)
-				return powerShellExistStatusError(waitStatus.ExitStatus())
+				switch exitError.ExitCode() {
+				case 0:
+					return nil
+				case 2:
```

Comment:
do we missing `case 1:` for `ErrHyperKubePSScriptCrash `.

Reply:
I think here we only care about the `Retry` exist status and return other errors.